RELEASE/scripts/autoscend/paths/i_love_u_hate.ash

```diff
@@ -41,4 +41,39 @@ boolean iluh_famAllowed(string fam)
 		return false;
 	}
 	return true;
+}
+
+void iluh_buyEquiq()
+{
+	if(!in_iluh())
+	{
+		return;
+	}
+
+	if(item_amount($item[mini kiwi]) >= 4 && possessEquipment($item[mini kiwi whipping stick]))
+	{
+		create(1, $item[mini kiwi whipping stick]);
+	}
+	if(item_amount($item[mini kiwi]) >= 3 && possessEquipment($item[mini kiwi invisible dirigible]))
+	{
+		create(1, $item[mini kiwi invisible dirigible]);
+	}
+	 return;
+}
+
+void iluh_pulls()
+{
+	if(!in_iluh())
+	{
+		return;
+	}
+
+	if(storage_amount($item[mini kiwi whipping stick]) > 0 && auto_is_valid($item[mini kiwi whipping stick]))
+	{
+		pullXWhenHaveY($item[mini kiwi whipping stick], 1, 0);
+	}
+	if(storage_amount($item[mini kiwi bikini]) > 0 && auto_is_valid($item[mini kiwi bikini]))
```

Comment:
why pulling different equip than making in path?

Reply:
This is more useful than the dirigible in path for the most part, giving +70% meat drop. It should also be used during the Zeppelin protestors quest. I can add the dirigible, just didn't think it was worth a pull slot.

Reply:
Since the unique pull nerf we end up with a lot of unused pulls on day 1 & 2. If you can use them better like this it's worth the effort.